src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -88,6 +90,16 @@ export const ConversationSpace: React.FunctionComponent = () => {
               )}
             </div>
           </div>
+          {!seatedAtTable && (
+            <TablesControlBar
+              className="control-bar"
+              defaultTables={defaultTables}
+              venue={venue}
+              users={recentVenueUsers}
+              updateTables={setTables}
```

Comment:
This is a callback and should be named something like `onUpdateTables`

Reply:
Actually, by looking at the implementation of `TablesControlBar` a better name would be `onToggleTables`

Reply:
fixed in f6a6e31bf722054e762d6e00c4ad34ed6e248279